src/main/java/com/kryptnostic/rhizome/pods/hazelcast/BaseHazelcastInstanceConfigurationPod.java

```diff
@@ -58,21 +62,64 @@ public Config getHazelcastServerConfiguration() {
             config
                     .setProperty( "hazelcast.logging.type", "slf4j" )
                     .setProperty( "hazelcast.slow.operation.detector.stacktrace.logging.enabled", "true" )
-                    .setProperty( "hazelcast.map.load.chunk.size","100000" )
+                    .setProperty( "hazelcast.map.load.chunk.size", "100000" )
                     .setClusterName( hzConfiguration.getGroup() )
                     .setSerializationConfig( serializationConfig() )
                     .setMapConfigs( mapConfigs() )
                     .setQueueConfigs( queueConfigs( config.getQueueConfig( defaultQueueName ) ) )
                     .setNetworkConfig( networkConfig( hzConfiguration ) );
 
+            hzConfiguration
+                    .getScheduledExecutors()
+                    .ifPresent( scheduledExecutors ->
+                            config.setScheduledExecutorConfigs( scheduledExecutorConfigs( scheduledExecutors ) ) );
+
+            hzConfiguration
+                    .getDurableExecutors()
+                    .ifPresent( durableExecutors ->
+                            config.setDurableExecutorConfigs( durableExecutorConfigs( durableExecutors ) ) );
+
             config.getCPSubsystemConfig().setCPMemberCount( hzConfiguration.getCPMemberCount() );
             return config;
         }
         return null;
     }
 
+    private Map<String, DurableExecutorConfig> durableExecutorConfigs( List<DurableExecutorConfiguration> durableExecutors ) {
+        return durableExecutors.stream()
+                .map( durableExecutor -> {
+                    final var dec = new DurableExecutorConfig( durableExecutor.getName() )
+                            .setPoolSize( durableExecutor.getPoolSize() )
+                            .setCapacity( durableExecutor.getCapacity() )
+                            .setDurability( durableExecutor.getDurability() );
+                    if ( durableExecutor.getSplitBrainProtectionName() != null &&
```

Comment:
Super minor readability preference for `StringUtils.isNotBlank( durableExecutor.getSplitBrainProtectionName() )`  (and below)